rfcs/convergence/apperance-migration.md

```diff
@@ -0,0 +1,106 @@
+# RFC: Appearance migration
+
+@peta-duda
+
+## Summary
+
+Fluent V9 changes the default appearance of input components (`Dropdown`, `Input`, `TextArea`). V0 input components default background color is grey-ish, whereas in V9 the background color is white.
+
+## Background
+
+Partners could change the background color to match the previous version of Fluent by passing a prop `appearance` with value `filled-darker`, but this solution does not scale well for partners when migrating to the new version and is error prone.
+
+## Problem statement
+
+This RFC explores potential solutions for partners, so they could migrate input components to V9 without the extra work of adding an additional prop to input components.
+
+Partners should also easily revert the decision to the default appearance value if they decide so, without changing inputs individually.
+
+👎 Cons of adding appearance to achieve the same design as V0:
+
+- Not scalable, partners would have to add the prop to every component
+- If they decide to revert back to default appearance in future, they would have to go through the same pain again by removing the prop
+- Is error prone
+
+## Detailed Design or Proposal
+
+### Composed component
+
+Partners could create a new composed component in fluent and modify the props in their preferred way. If the partner would like to keep the original color, they could create the composed component and have the default component without the appearance prop renders as `filled-darker`.
+
```

Comment:
````suggestion

#### Example

```ts
export const Input: ForwardRefComponent<InputProps> = React.forwardRef((props, ref) => {
  const state = useInput_unstable({ appearance: 'filled-darker', ...props }, ref);

  useInputStyles_unstable(state);
  return renderInput_unstable(state);
});
```
````

Can we add an example to the option? (if I understood it properly)